build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
 	compileOnly 'org.projectlombok:lombok'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	implementation("org.springdoc:springdoc-openapi-ui:1.6.12")
```

Comment:
let's do the same as we did with maven - move library versions
```
buildscript {
    ext {
        openApiVersion = "1.6.12"
    }
}
```
Important! This build script must be at the beginning of `build.gradle` file, otherwise it will not work

and make dependency like this
`implementation "org.springdoc:springdoc-openapi-ui:${openApiVersion}"`

Reply:
Ok, moved the `openApiVersion ` version this commit. Is an `ext {}` block required for each version or can all be added there?